dd-sdk-android-ktx/src/main/kotlin/com/datadog/android/ktx/tracing/ClosableExt.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.ktx.tracing
+
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.RumErrorSource
+import java.io.Closeable
+
+internal const val REALM_ERROR_MESSAGE = "Error while using the closeable"
```

Comment:
This should be a `CLOSEABLE_ERROR_MESSAGE`

Reply:
Oups...wrong copy paste